backend/nokia-innovative-project/src/main/java/com/nokia/library/nokiainnovativeproject/controllers/PictureUploadController.java

```diff
@@ -0,0 +1,29 @@
+package com.nokia.library.nokiainnovativeproject.controllers;
+
+
+import com.cloudinary.Cloudinary;
+import com.cloudinary.utils.ObjectUtils;
+import com.nokia.library.nokiainnovativeproject.services.PictureService;
+import com.nokia.library.nokiainnovativeproject.utils.Mappings;
+import com.nokia.library.nokiainnovativeproject.utils.MessageInfo;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Map;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(Mappings.API_VERSION + Mappings.PICTURES)
+public class PictureUploadController {
+
+    private final PictureService pictureService;
+
+    @PostMapping(value = "/upload")
+    public MessageInfo uploadPicture(MultipartFile picture) throws IOException {
```

Comment:
It would be nice to specify somehow max file size.

Reply:
I think this should be handled on client side.

Reply:
In FE, file size should be checked before sending the request.

Reply:
Yes, file size should be checked before sending but it should be checked on server side too. 
There is possibility to upload file via API.